tests/integration/test_authd/data/force_options.yaml

```diff
@@ -52,7 +52,7 @@
       input: "OSSEC A:'nn'"
```

Comment:
The `insert_prev_agent` allows the testing of the force option, because it is used to replace or not a previous agent. All these tests already have been grouped in this file because they test the same feature.